spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/LibraryControllerHelperService.java

```diff
@@ -359,164 +329,6 @@ public JSONObject changeLibraryIdBarcode(HttpSession session, JSONObject json) {
     return JSONUtils.SimpleJSONResponse("New identification barcode successfully assigned.");
   }
 
-  public JSONObject getLibraryDilutionBarcode(HttpSession session, JSONObject json) {
```

Comment:
I'm hesitant to remove this entirely, as there's no other place to get the dilution matrix barcode (the dilutions table on the Edit Library page has been changed in this PR to no longer display the matrix barcode at all) and I don't know if EI prints theirs from this location. Perhaps ask them? Or add in a "Print barcodes" option in the bulk menu?